app/src/main/java/org/mozilla/focus/utils/ViewUtils.java

```diff
@@ -104,12 +104,14 @@ public static void showKeyboard(View view) {
     }
 
     public static void hideKeyboard(View view) {
-        InputMethodManager imm = (InputMethodManager) view.getContext().getSystemService(Context.INPUT_METHOD_SERVICE);
-        if (imm == null) {
-            return;
-        }
+        if (view != null) {
+            InputMethodManager imm = (InputMethodManager) view.getContext().getSystemService(Context.INPUT_METHOD_SERVICE);
+            if (imm == null) {
+                return;
+            }
```

Comment:
In our code we mostly prefer early returns (like with imm == null).

Reply:
I'll just ammend this to the previous commit